support/CMakeLists.txt

```diff
@@ -152,15 +152,3 @@ add_custom_target(uninstall-rootfs-image-module
   COMMAND ${CMAKE_COMMAND} -D CMAKE_INSTALL_COMPONENT=rootfs-image-module -P ${CMAKE_BINARY_DIR}/cmake_uninstall.cmake
   WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
 )
-
-install(FILES ${CMAKE_SOURCE_DIR}/support/version
```

Comment:
Q: Why not just revert the commit?

Reply:
Actually, I didn't notice that this was done in one commit. And indeed I missed the removal of the `version` file itself. I'll revert what I did, and then revert the original instead. 😄

Reply:
Not strictly needed ofc, but do as you see best :)